source/framework/CMakeLists.txt

```diff
@@ -11,6 +11,14 @@ file(GLOB_RECURSE addon_src
         "tiny*cpp"
         "startup.cpp")
 
+if (NOT ${REST_EVE} MATCHES "ON")
```

Comment:
This needs to be here, because if not `TRestComplex` was not properly excluded. I think because the excludes variable was not transferred properly to `COMPILEDIR`.

Reply:
If not remove `mpfr` libraries from the system and try `-DREST_MPFR=OFF`. It will still try to compile `TRestComplex.

See also https://root-forum.cern.ch/t/question-about-how-to-cast-in-pyroot/53116/20